code401-challenges/src/main/java/code401/challenges/quickSort/QuickSort.java

```diff
@@ -0,0 +1,136 @@
+package code401.challenges.quickSort;
+
+import java.util.Arrays;
+
+public class QuickSort {
+
+    public void quickSortFunc(int[] arr, int left, int right)
+    {
```

Comment:
In Java it is standard practice to put your opening `{` on the same line as the method definition/conditional/loop